Dockerfiles/Dockerfile.dela

```diff
@@ -2,12 +2,16 @@ FROM golang:1.20.6-bookworm
 
 RUN apt-get update && apt-get install git
 RUN git clone https://github.com/dedis/dela.git
-RUN git clone https://github.com/dedis/d-voting.git
+WORKDIR /go/dela
+# make sure we're using the same head as d-voting
+RUN git checkout fix-bbolt
 WORKDIR /go/dela/cli/crypto
 RUN go install
+WORKDIR /go/d-voting
+COPY .. .
```

Comment:
Surprised question: since when can you copy the parent directory in Dockerfiles?

Reply:
arf, that's a typo from a left-over file path, it doesn't seem to behave differently from `COPY . .`, I'll fix that